keps/sig-api-machinery/20190228-priority-and-fairness.md

```diff
@@ -745,11 +756,12 @@ spec:
   queueLengthLimit: 1000
 ```
 
-For user requests from kubectl.
+For normal workloads:
```

Comment:
The original intent here was to distinguish user requests issued using kubectl from all other normal workload requests.  This revision replaces (A) "user requests from kubectl" vs "requests from controllers processing workload" with (B) "normal workloads" vs "requests from controllers processing workload".  I think that "requests from controllers processing workload" is part of "normal workloads", so that makes the wording for (B) problematic.  Also, this revision replaces two names that include "workload" with one name that is simply "workload" and another that does not include "workload" but is associated with a meaning that involves "workload".  If we are going to insist on one name that does not include the word "workload" but has a meaning that *does* involve workload then I suggest the other name be one that identifies its distinction (e.g., "workload-controllers").  This revision also moves much of the workload to "system-low"; in the original, "system-low" was deliberately very restricted, to just the garbage collector, and the other controllers were assigned to "workload-low".

Reply:
i renamed all default objects. now they're:

1.  FS `exempt` => PL `exempt`
2. FS `nodes`, `leader-election` => PL `system-high`
3. FS `kube-controller-manager`, `kube-scheduler` => PL `system-low`
4. FS `serviceaccounts` => PL `workload`
5. FS `default` => PL `default`

is it cleaner ?

Reply:
Did another review to respond.